src/config.rs

```diff
@@ -19,15 +32,21 @@ impl Default for TermvizConfig {
         let conf= TermvizConfig {
             fixed_frame: "map".to_string(),
             camera_frame: "base_link".to_string(),
-            map_topics: vec!["map".to_string()],
-            laser_topics: vec!["scan".to_string()],
-            marker_array_topics: vec!["marker_array".to_string()],
-            target_framerate: 42,
+            map_topics: vec![ ListenerConfig {
+                topic: "map".to_string(),
+                color: Color { r: 255, b: 255, g: 255 } }],
+            laser_topics: vec![ListenerConfig {
+                topic: "scan".to_string(),
+                color: Color { r: 200, b: 0, g: 0 } }],
+            marker_array_topics: vec![ListenerConfig {
+                topic: "marker_array".to_string(),
+                color: Color { r: 20, b: 20, g: 200 } }],
```

Comment:
color for markers is not having an effect right now, its using the Colors from markers